pom-template.xml

```diff
@@ -201,10 +201,10 @@
     <dependency><groupId>commons-fileupload</groupId><artifactId>commons-fileupload</artifactId><scope>provided</scope></dependency>
     <dependency><groupId>commons-io</groupId><artifactId>commons-io</artifactId><scope>provided</scope></dependency>
     <dependency><groupId>commons-lang</groupId><artifactId>commons-lang</artifactId><scope>provided</scope></dependency>
+    <dependency><groupId>dev.zio</groupId><artifactId>izumi-reflect-thirdparty-boopickle-shaded_${scala-binary-version}</artifactId><scope>provided</scope></dependency>
```

Comment:
Why do we need to adjust dependencies on plugins-common ?

Reply:
they changed some dependencies when switching to zio `RC18-2`. 
Maven don't manage the provided one correctly if we don't tell him like that, so it may put them in the rpkg (because it doesn't see that they are provided), which lead to bigger rpkg (and horror stories in case of version mismatch)